gemfirexd/tools/src/dunit/java/com/pivotal/gemfirexd/transactions/SnapShotTxDUnit.java

```diff
@@ -57,6 +57,18 @@ private static File getDiskDir() {
     return dir;
   }
 
+  public static void validateNoActiveSnapshotTx() {
```

Comment:
You can have this method in the base class and avoid this duplication. Isn't it?